S23/pbutiya/CodeReview.py

```diff
@@ -0,0 +1,118 @@
+print("")
+print("**************************************************************WELCOME TO THE UTILITY*********************************************")
+print("****************************************************Convert UPPERCASE Into lowercase*********************************************")
+print("*********************************************************************************************************************************")
+print("")
+# Creating an array for Langauge List, In future more langauges can get added to array.
+languageList = ["ENGLISH", "TURKISH or AZERBAIJANI", "IRISH","GREEK"]
+
+# Function to select from the List
+def language_Function():
+    print("")
+    print("****************************************************************PLEASE SELECT LANGAUGE **********************************")
+    print("********* 1."+languageList[0])
+    print("********* 2."+languageList[1])
+    print("********* 3."+languageList[2])
+    print("********* 4."+languageList[3])
+    print("")
+    print("************************************************ NOTE: USE THE SERIAL NUMBER TO SELECT A LANGAUGE************************")
+    print("-------------------------------------------------For example : Press 1 for English---------------------------------------")
+    print("")
+    select_Function()
+    
+# Logic for each langauge, In future more langauges can be added and their logic can be written.
+def select_Function():
+    inputString = input("ENTER OPTION FROM ABOVE LIST --->")
+    
+    if inputString=="1": # For ENGLISH
+        print("************************************ ******** LANGAUGE SELECTED IS - "+languageList[0]+" ********************************************" )
+        print("")
+        inputString = input("PLEASE ENTER TEXT ---> ")
+        inputString=inputString.lower() 
+        print("")
+        print("LOWERCASE OUTPUT ---> " + inputString)
+        exit_Function()
+    
+    
+    if inputString=="2": # For TURKISH OR AZERBAIJANI
```

Comment:
It is better to use elif statements, as this will improve the performance of the code.